backend/app/controllers/api/v1/accounts/projects_controller.rb

```diff
@@ -12,6 +12,8 @@ def create
               language: current_user.account.language
             )
           )
+          Translations::TranslateProjectJob.perform_later(project.id)
```

Comment:
I believe it makes probably more sense to add this as after action on model -- at least if we plan to run translation job everytime when translatable column changes. If this approach would be too aggressive, it could be moved to some controller concern and again as after action. What I am trying to avoid is situation where you have to explicitly define translation across all endpoints <-- much easier and maintainable is to have it at one place for all translatable models

Reply:
OK, that's what I did first, but then found specs irritating to update and ended up with this :)